### DISCUSSION TITLE
Unable to install chainlink-brownie-contracts with forge install

### DISCUSSION THREAD

**Discussion by SamyakL** [Cyfrin/foundry-full-course-cu#2277] [Q&A] [answered]
(url: Some("https://github.com/smartcontractkit/chainlink-brownie-contracts"), tag: Some("1.1.1"))
fatal: not a git repository (or any of the parent directories): .git
Error: 
git submodule exited with code 128

**Comment by EngrPips** [comment]
What command did you run @SamyakL

**Reply by SamyakL** [reply]
forge install smartcontractkit/chainlink-brownie-contracts@1.1.1 --no-commit

**Reply by SamyakL** [reply]
copied from the ofc doc, and just added the pinned dependency, tried it without the pinned dependency as well, that didnt work out either

**Reply by EngrPips** [reply]
Hello @SamyakL, It worked for me. 

<img width="1440" alt="Screenshot 2024-08-20 at 12 14 20" src="https://github.com/user-attachments/assets/075785c6-7f7e-4f19-88a4-8de7d0d83d5c">

I assume you didn't initialize a foundry repo before you ran the command, and that is why it didn't work

**Reply by EngrPips** [reply, marked as answer]
Inside the repo you want your project in, you should first run `forge init.` Then you can use the command `forge install smartcontractkit/chainlink-brownie-contracts@1.1.1 --no-commit`, and it is supposed to work
